lib/ansible/modules/storage/netapp/na_ontap_qtree.py

```diff
@@ -81,26 +101,31 @@
 
 """
 import traceback
-
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils._text import to_native
 import ansible.module_utils.netapp as netapp_utils
-
+from ansible.module_utils.netapp_module import NetAppModule
 
 HAS_NETAPP_LIB = netapp_utils.has_netapp_lib()
 
 
 class NetAppOntapQTree(object):
+    '''Class with qtree operations'''
 
     def __init__(self):
         self.argument_spec = netapp_utils.na_ontap_host_argument_spec()
         self.argument_spec.update(dict(
-            state=dict(required=False, choices=[
-                       'present', 'absent'], default='present'),
+            state=dict(required=False,
+                       choices=['present', 'absent'],
+                       default='present'),
             name=dict(required=True, type='str'),
             from_name=dict(required=False, type='str'),
             flexvol_name=dict(type='str'),
```

Comment:
required=False  

I'm not sure it's needed, but it is present for other parameters.

Reply:
It's not needed, by default is "required=False".  I didn't see any best practice about specifying always or not.